client/components/jetpack/portal-nav/index.tsx

```diff
@@ -53,6 +53,12 @@ export default function PortalNav( { className = '' }: Props ) {
 	const isPartnerPortalRoute = useSelector( isPartnerPortal );
 	const selectedSiteId = useSelector( getSelectedSiteId );
 	const manageSiteLink = useManageSiteLink();
+
+	usePresalesChat(
+		'jpAgency',
```

Comment:
Fully agree, but it feels like a losing battle against what seems to be the convention. A compromise could be introducing these strings as named constants somewhere, so that we can take advantage of the enum behaviour without "magic" strings around. I'll keep that in mind for follow-up patches 🙇‍♂️